include/spirv-tools/optimizer.hpp

```diff
@@ -167,6 +167,10 @@ Optimizer::PassToken CreateUnifyConstantPass();
 // OpSpecConstantOp.
 Optimizer::PassToken CreateEliminateDeadConstantPass();
 
+// Creates a inline pass.
+// An inline pass exhaustively inlines all function calls.
+Optimizer::PassToken CreateInlinePass();
```

Comment:
Done. Indeed, this pass only performs inlining on functions which are marked as entry points. More judicious application of inlining is left for future work.